### PR TITLE
ID-356 Converted px to rem for font-sizes

### PR DESCRIPTION
**Pull request by WarwickGoatee**

### CODE REVIEW

**Comment by WarwickGoatee** [comment]
IE11 won't resize the text for some reason. Otherwise seems to be okay?

**Comment by lol768** [comment]
>IE11 won't resize the text for some reason.

Given even MS Teams is dropping support for it this quarter, not sure we should be too concerned.

**Comment by WarwickGoatee** [comment]
> > IE11 won't resize the text for some reason.
> 
> Given even MS Teams is dropping support for it this quarter, not sure we should be too concerned.

Yeah I don't really care either, just noting here for posterity really.

**Review by lol768 — APPROVED** [review]

File: docs/assets/id6/css/fonts/standard.css [hunk 1]
```diff
@@ -62,7 +62,7 @@ var {
 }
 pre,
 code {
-  line-height: 16px;
+  line-height: 1.6rem;
```
Comment (matmannion) [inline docs/assets/id6/css/fonts/standard.css:65]: This is in ID6 and can be reverted

**Review by matmannion — APPROVED** [review]

**Pull request merged**